File: blog-post-2/kotlin/StudentMarks.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var scanner = Scanner(System.`in`)
    print("Enter Math mark: ")
    var mMark = scanner.nextInt()
    print("Enter Science mark: ")
    var sMark = scanner.nextInt()
    print("Enter English mark: ")
    var eMark = scanner.nextInt()

    var validate = false;

    if ((mMark >= 0 && mMark <= 100) 
            && (mMark >= 0 && mMark <= 100) 
            && (mMark >= 0 && mMark <= 100)) {

        validate = true

    }
    
    if (validate) {
        var average = (mMark + sMark + eMark) / 3
        if (average < 30) {
            println("Fail")
        } else if (average < 40) {
            println("Grade: C")
        } else if (average < 60) {
            println("Grade: B")
        } else {
            println("Grade: A")
        }
    }
    else{
        println("Wrong input")
    }
}

File: blog-post-2/python/StudentMarks.py
mMark = int(input("Enter Math mark: "))
sMark = int(input("Enter Science mark: "))
eMark = int(input("Enter English mark: "))

validate = False

if (mMark >= 0 and mMark <= 100
    and (mMark >= 0 and mMark <= 100)
    and (mMark >= 0 and mMark <= 100)) :
        validate = True

if (validate):
    average = (mMark + sMark + eMark) / 3
    if (average < 30):
        print("Fail")
    elif (average < 40):
        print("Grade: C")
    elif (average < 60):
        print("Grade: B")
    else:
        print("Grade: A")


else:
    print("Wrong input")
